### PR TITLE
DOC-1662 Diagnostic updates.

### PR DESCRIPTION
**Pull request by stbof**
https://cockroachlabs.atlassian.net/browse/DOC-1662

### CODE REVIEW

**Review by RaduBerinde — APPROVED** [review]
LGTM, thanks!

File: v21.2/cockroach-statement-diag.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Outstanding activation requests:
   603811900498804737  2020-11-02 17:46:08 UTC  SELECT * FROM bank.accounts
 ~~~
 
-Delete an activation request:
+### Delete an activation request:
```
Comment (rmloveland) [inline v21.2/cockroach-statement-diag.md:135]: nit: don't need colon on header

File: _includes/v21.2/sql/shell-commands.md [hunk 2]
```diff
@@ -2,20 +2,22 @@ The following commands can be used within the interactive SQL shell:
...
 `\l` | List all databases in the CockroachDB cluster. This command is equivalent to [`SHOW DATABASES`](show-databases.html).
-`\dt`<br><br>`d` | Show the tables of the current schema in the current database. These commands are equivalent to [`SHOW TABLES`](show-tables.html).
+`\dt`,`d` | Show the tables of the current schema in the current database. These commands are equivalent to [`SHOW TABLES`](show-tables.html).
 `\dT` |  Show the [user-defined types](enum.html) in the current database. This command is equivalent to [`SHOW TYPES`](show-types.html).
 `\du` | List the users for all databases. This command is equivalent to [`SHOW USERS`](show-users.html).
 `\d <table>` | Show details about columns in the specified table. This command is equivalent to [`SHOW COLUMNS`](show-columns.html).
-`\r` |   Resets the query input buffer, clearing all SQL statements that have been entered but not yet executed.
+`\r` | Resets the query input buffer, clearing all SQL statements that have been entered but not yet executed.
+`\statement-diag list`   |  List available diagnostic bundles.
```
Comment (rmloveland) [inline _includes/v21.2/sql/shell-commands.md:19]: I think these two `statement-diag` commands would be a bit more helpful if 'diagnostic bundle' linked to https://www.cockroachlabs.com/docs/dev/explain-analyze.html#explain-analyze-debug  or perhaps https://cockroachlabs.com/docs/v21.2/cockroach-statement-diag.html

**Review by rmloveland — APPROVED** [review]
LGTM, little comments

**Pull request merged**